ts/features/messages/components/MessageDetail/MessageTitle.tsx

```diff
@@ -1,24 +1,8 @@
-import * as React from "react";
+import React from "react";
 import { H1 } from "../../../../components/core/typography/H1";
-import I18n from "../../../../i18n";
-import { Body } from "../../../../components/core/typography/Body";
 
-type Props = {
+type MessageTitleProps = {
   title: string;
-  isPrescription: boolean;
 };
 
-/**
- * Render a title for a message
- * @param props
- * @constructor
- */
-export const MessageTitle = (props: Props): React.ReactElement =>
-  props.isPrescription ? (
-    <>
-      <H1>{I18n.t("messages.medical.prescription")}</H1>
-      <Body>{I18n.t("messages.medical.memo")}</Body>
-    </>
-  ) : (
-    <H1>{props.title}</H1>
-  );
+export const MessageTitle = ({ title }: MessageTitleProps) => <H1>{title}</H1>;
```

Comment:
Should we delete this component and move the `<H1>{title}</H1>` into the component that is referencing it?

Reply:
Applied https://github.com/pagopa/io-app/pull/5407/commits/f421d75cbfcbde91d2ce52b6952e61d9e93e8a92